.github/workflows/chain_interaction.yml

```diff
@@ -63,7 +63,9 @@ jobs:
         working-directory: ./packages/bindings-test
         run: | 
           ../../desmos/spawn_test_chain.sh -b
-          sleep 2 
+          sleep 2
+          ../../desmos/upload_test_contract.sh
+          cargo run
```

Comment:
Since the previous setup_chain.sh includes legacy version msgs, so we need to initialize the msgs in startgate version by rust script instead.